.gitattributes

```diff
@@ -11,4 +11,4 @@ test/snapshots/**/*.txt linguist-generated
 # Preserve \n on git clone on Windows, regardless of the git config core.autocrlf value
 # See https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings
 **/*.rb text eol=lf
-**/*.txt text eol=lf
```

Comment:
I think this removes all `\r\n` on checkout. Let me know if there is a different configuration I am not understanding.

Reply:
Ah, looks like you already did this on main, but better.